docs/samples/shards/UI/Table/1.edn

```diff
@@ -0,0 +1,37 @@
+(defloop main-wire
+  (GFX.MainWindow
+   :Contents
+   (->
+    (Setup
+     (GFX.DrawQueue) >= .ui-draw-queue
+     (GFX.UIPass .ui-draw-queue) >> .render-steps)
+    .ui-draw-queue (GFX.ClearQueue)
+
+    (UI
+     .ui-draw-queue
+     (UI.CentralPanel
+      :Contents
+      (->
+       [{:Name "Doe" :Surname "John"}
+        {:Name "Dough" :Surname "Jane"}
+        {:Name "Smith" :Surname "Dick"}]
+       (UI.Table
+        :Resizable true
+        :Striped true
+        :RowIndex .index
+        :Columns
+        [{:Initial (float 20)}
+         {:Header "Surname"
```

Comment:
Also maybe header should be a callback too, sometimes you might want different styling, maybe an image/icon.

Reply:
Header callback sounds good. I will make it support either a string (keep current behavior) or a shard.